<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationWindow</class>
 <widget class="QMainWindow" name="ConfigurationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigurationWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit_tlb">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Anzahl Einträge</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QLabel" name="label_pagingAlgorithmus">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Paging Algorithmus:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QPushButton" name="pushButton_loadConfigs">
        <property name="text">
         <string>Lade Konfiguration</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QLabel" name="label_tlb">
        <property name="text">
         <string>Größe TLB:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_page">
        <property name="text">
         <string>Seitenrahmen/virtuele Seite:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_pagingAlgorithmus">
        <property name="currentText">
         <string>Not Recently Used</string>
        </property>
        <property name="maxVisibleItems">
         <number>10</number>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>Not Recently Used</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>First In First Out</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Second Chance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Least Recently
Used</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Not Frequently Used (mit Aging)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Not Frequently Used (ohne Aging)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_physicalStorage">
        <property name="text">
         <string>Physikalischer Speicher: </string>
        </property>
       </widget>
      </item>
      <item row="8" column="2" colspan="2">
       <widget class="QLabel" name="label_countPages">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Anzahl virtuelle Seiten: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLabel" name="label_virtualStorage">
        <property name="text">
         <string>Virtueller Speicher (pro Prozess):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_page">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Bytes</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_physicalStorage">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Bytes</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit_virtualStorage">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Bytes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label_title">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Paging Simulator - Konfiguation</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLabel" name="label_countFrames">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Anzahl Seitenrahmen:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
